client/state/action-types.js

```diff
@@ -10,6 +10,9 @@
  *  - https://wpcalypso.wordpress.com/devdocs/docs/our-approach-to-data.md#actions
  */
 
+export const ANALYTICS_BUMP_STAT = 'ANALYTICS_BUMP_STAT';
+export const ANALYTICS_RECORD_EVENT = 'ANALYTICS_RECORD_EVENT';
+export const ANALYTICS_RECORD_PAGE_VIEW = 'ANALYTICS_RECORD_PAGE_VIEW';
```

Comment:
>is this really applicable? To me it reads funny

I'd side with @gwwar on this one, mostly for consistency and that we don't know what other action types we might incorporate in the future, e.g. more actions related to page view than just record? In that case, grouping action types by `ANALYTICS_PAGE_VIEW_RECORD`, `ANALYTICS_PAGE_VIEW_FOO` lends to our aim in making the action type list more easily scannable.